### PR TITLE
Use GitHub actions

### PR DESCRIPTION
**Pull request by MancunianSam**

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -0,0 +1,27 @@
+name: TDR Tag and pre deploy
```
Comment (TomJKing) [inline .github/workflows/build.yml:1]: Maybe add `Auth Server` to the name?

**Review by TomJKing — COMMENTED** [review]
Looks good. One very minor comment.

Did you test the SPIs when they were built together? I seem to vaguely remember trying this and having some problems with the build jars, but I could be wrong?

**Comment by MancunianSam** [comment]
I've built the docker image locally with newly built versions of the SPI jars and the server starts and everything seems ok.

**Review by TomJKing — APPROVED** [review]
Looks good. Thanks

**Pull request merged**